src/op_analytics/transforms/fees/update/04_agg_daily_trace_to.sql

```diff
@@ -0,0 +1,257 @@
+
+  , SUM(tx_l1_fee_native) AS sum_tx_l1_fee_native
+  , sumIf(tx_l1_fee_native, tx_success) AS sum_tx_l1_fee_native_tx_success
+
+  , SUM(tx_l2_fee_native) AS sum_tx_l2_fee_native
+  , sumIf(tx_l2_fee_native, tx_success) AS sum_tx_l2_fee_native_tx_success
+
+  , SUM(tx_l2_priority_fee_native) AS sum_tx_l2_priority_fee_native
+  , sumIf(tx_l2_priority_fee_native, tx_success) AS sum_tx_l2_priority_fee_native_tx_success
+
+  , SUM(tx_l2_base_fee_native) AS sum_tx_l2_base_fee_native
+  , sumIf(tx_l2_base_fee_native, tx_success) AS sum_tx_l2_base_fee_native_tx_success
+
+  , SUM(tx_l2_legacy_extra_fee_native) AS sum_tx_l2_legacy_extra_fee_native
+  , sumIf(tx_l2_legacy_extra_fee_native, tx_success) AS sum_tx_l2_legacy_extra_fee_native_tx_success
+
+  -- Transaction-Level Gas Prices
+  , avgWeighted(tx_l2_gas_price_gwei, tx_l2_gas_used) AS avg_tx_l2_gas_price_gwei_called
+  , avgWeighted(tx_l2_priority_gas_price_gwei, tx_l2_gas_used) AS avg_tx_l2_priority_gas_price_gwei_called
+  , avgWeighted(tx_l2_legacy_extra_gas_price_gwei, tx_l2_gas_used) AS avg_tx_l2_legacy_extra_gas_price_gwei_called
+  , avgWeighted(tx_l1_base_gas_price_gwei, tx_l2_gas_used) AS avg_tx_l1_base_gas_price_gwei_called
+  , avgWeighted(tx_l1_blob_base_gas_price_gwei, tx_l2_gas_used) AS avg_tx_l1_blob_base_gas_price_gwei_called
+
+
+  -- Transaction-Level transaction sizes
+  , SUM(tx_input_byte_length) AS sum_tx_input_byte_length_called
+  , SUM(tx_input_zero_bytes) AS sum_tx_input_zero_bytes_called
+  , SUM(tx_input_nonzero_bytes) AS sum_tx_input_nonzero_bytes_called
+  , SUM(tx_l1_base_scaled_size) AS sum_tx_l1_base_scaled_size_called
+  , SUM(tx_l1_blob_scaled_size) AS sum_tx_l1_blob_scaled_size_called
+  , SUM(tx_l1_gas_used_unified) AS sum_tx_l1_gas_used_unified_called
+  , SUM(tx_estimated_size) AS sum_tx_estimated_size_called
+
+  -- Sum Internal Trace Gas
+  /*
+    Note: As opposed to transaction-level equivalent metrics, these CAN be summed up at a project level.
+    These methodologies were designed to distribute gas usage and fees across each call, so that they
+    still represent a "part of a whole," whereas transaction-level metrics do not.
+  */
+
+  , SUM(preagg_sum_trace_gas_used) AS sum_trace_gas_used
+  , sumIf(preagg_sum_trace_gas_used, tx_success) AS sum_trace_gas_used_tx_success
+
+  , SUM(preagg_sum_trace_gas_used_minus_subtraces) AS sum_trace_gas_used_minus_subtraces
+  , sumIf(preagg_sum_trace_gas_used_minus_subtraces, tx_success) AS sum_trace_gas_used_minus_subtraces_tx_success
+
+  , SUM(preagg_sum_tx_l2_gas_used_amortized_by_call) AS sum_tx_l2_gas_used_amortized_by_call
+  , sumIf(preagg_sum_tx_l2_gas_used_amortized_by_call, tx_success) AS sum_tx_l2_gas_used_amortized_by_call_tx_success
+
+  , SUM(preagg_sum_tx_l1_gas_used_unified_amortized_by_call) AS sum_tx_l1_gas_used_unified_amortized_by_call
+  , SUM(preagg_sum_tx_l1_base_scaled_size_amortized_by_call) AS sum_tx_l1_base_scaled_size_amortized_by_call
```

Comment:
we can cut `preagg_sum_tx_l1_base_scaled_size_amortized_by_call` and `preagg_sum_tx_l1_blob_scaled_size_amortized_by_call` metrics here